app/views/content/life-as-a-teacher/pay-and-benefits/abigails-favourite-things-about-teaching/_header.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render Content::LandingHeroComponent.new(
+  title: @front_matter ["title"],
```

Comment:
We should remove the spaces on lines 2 and 3 from `@front_matter ["title"]` to `@front_matter["title"]` to make them consistent with the ones on line 4 and 5